File: 9_PalindromeNumber/unoptimal.c
bool isPalindrome(int x) {
    if(x < 10){
        return x >= 0;
    }
    if(x % 10 == 0){
        return false;
    }
    int arr[10];
    int n = 0;
    while(x != 0){
        arr[n] = x % 10;
        n++;
        x /= 10;
    }
    int j = n-1;
    for(int i = 0; i < (n/2); i++){
        if(arr[i] != arr[j]){
            return false;
        }
        j--;
    }
    return true;
}

File: 1018_BinPrefixDivBy5/unoptimal.py
class Solution(object):
    def prefixesDivBy5(self, nums):
        arr = []
        x = 0
        i = 0
        for i in range(len(nums)):
            if(nums[i] == 1):
                x = (x*2) + 1
            else:
                x *= 2
            if(x % 5 == 0):
                arr.append(True)
            else:
                arr.append(False)
        return arr

        """
        :type nums: List[int]
        :rtype: List[bool]
        """
        
